pkg/spdk/backing_image.go

```
@@ -487,11 +490,13 @@
 		return errors.Wrapf(err, "failed to get the lvs name for backing image %v with lvs uuid %v", bi.Name, bi.LvsUUID)
 	}
 	bi.LvsName = lvsName
-	bi.log = logrus.StandardLogger().WithFields(logrus.Fields{
+
+	bi.log.UpdateLogger(logrus.Fields{
```

golangci-lint (1.62.2)

Error return value of `bi.log.UpdateLogger` is not checked

(errcheck)